src/main/java/gift/dto/MemberRequest.java

```diff
@@ -0,0 +1,6 @@
+package gift.dto;
+
+import jakarta.validation.constraints.Email;
+
+public record MemberRequest(@Email(message = "이메일 형식의 입력이어야 합니다.") String email, String password) {
```

Comment:
```suggestion
public record MemberRequest(
@Email(message = "이메일 형식의 입력이어야 합니다.") String email, 
String password
) {
```

한줄로 적기보단 개행을 두면 훨씬 더 가독성이 높아질거 같습니다.

Reply:
저도 가독성이 부족한 것 같다는 생각을 했는데, 개행을 넣으면 되는군요! 수정하겠습니다.